gtsfm/two_view_estimator.py

```diff
@@ -195,11 +215,18 @@ def check_for_degeneracy(
         """ """
         insufficient_inliers = two_view_report.num_inliers_est_model < self._min_num_inliers_acceptance
 
+        H_EF_inlier_ratio = two_view_report.num_H_inliers / (two_view_report.num_inliers_est_model + EPSILON)
+        is_planar_or_panoramic = H_EF_inlier_ratio > MAX_H_INLIER_RATIO
```

Comment:
`MAX_H_INLIER_RATIO` should be renamed? Why not `MAX_H_TO_EPIPIPOLAR_INLIER_RATIO`?